npeps/npep-173-service-account-matches.md

```diff
@@ -0,0 +1,99 @@
+# NPEP-173: NPEP template
+
+* Issue: [#173](https://github.com/kubernetes-sigs/network-policy-api/issues/173)
+* Status: Provisional
+
+## TLDR
+
+Ability to create policies that control network traffic based on workload identities a.ka.
+[Kubernetes service accounts](https://kubernetes.io/docs/concepts/security/service-accounts/).
+
+## Goals
+
+* Use service accounts (identities) as a way of describing (admin) network policies for pods
+
+## Non-Goals
+
+* There might be other identity management constructs like SPIFFE IDs which
```

Comment:
It's not impossible, perhaps, that we could linke SPIFFE IDs to service accounts, at least superfiailly?  It would not be crypto-strong but it might indicate that someone with elevated authority (status) has asserted that namespace "foo", ServiceAccount "bar" has the same intended scope as SPIFFE "mycompany.com/ns/foo/id/bar" or whatever?

This is a bit outside my domain, maybe it's a terrible idea if you have more than half a clue..